app/models/invenio_rdm_record_converter.rb

```diff
@@ -46,6 +49,11 @@ def initialize(generic_file=nil)
     # @permissions = permissions(generic_file)
   end
 
+  def to_json(options={})
```

Comment:
The memoized header lookups are in a member variable so when `#to_json` is called it would include those in the return value. This keeps those values out.

Reply:
No problem, it was really odd to me that this was the standard way of doing this. At least it seemed to be the standard.